File: applications/event-account-http-source/src/main/java/showcase/event/stream/rabbitmq/account/http/source/AccountHttpSourceApp.java
package showcase.event.stream.rabbitmq.account.http.source;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class AccountHttpSourceApp {

	public static void main(String[] args) {

		System.out.println("ENV="+System.getenv()+ " PROPERTIES"+System.getProperties());

		SpringApplication.run(AccountHttpSourceApp.class, args);
	}

}

File: applications/stream-account-http-source/src/main/kotlin/com/vmware/tanzu/data/services/rabbitmq/streaming/account/AccountHttpSourceApp.kt
package com.vmware.tanzu.data.services.rabbitmq.streaming.account

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class AccountHttpSourceApp

fun main(args: Array<String>) {
	runApplication<AccountHttpSourceApp>(*args)
}
